Sources/PackageModel/Manifest/TargetDescription.swift

```diff
@@ -24,15 +24,15 @@ public struct TargetDescription: Equatable, Codable {
     /// Represents a target's dependency on another entity.
     public enum Dependency: Equatable {
         case target(name: String, condition: PackageConditionDescription?)
-        case product(name: String, package: String?, condition: PackageConditionDescription?)
+        case product(name: String, moduleAliases: [String: String]? = nil, package: String?, condition: PackageConditionDescription?)
```

Comment:
I think the order should be name, package, moduleAliases, condition to be consistent with the comment above